docs/src/main/tut/freemonad.md

```diff
@@ -166,14 +166,14 @@ DSL. By itself, this DSL only represents a sequence of operations
 To do this, we will use a *natural transformation* between type
 containers.  Natural transformations go between types like `F[_]` and
 `G[_]` (this particular transformation would be written as
-`NaturalTransformation[F,G]` or as done here using the symbolic
+`FunctionK[F,G]` or as done here using the symbolic
 alternative as `F ~> G`).
```

Comment:
That does make sense, yes. It felt weird to use `FunctionK` in the documentation, that's why I left the documentation itself untouched.